coalib/coala.py

```diff
@@ -27,6 +29,8 @@
 
 
 def main():
+    logging.config.dictConfig(get_log_config_dict())
```

Comment:
maybe that function should actually do the dictConfig invocation as well - less duplication cross all those files